src/test/java/com/egov/business-services/tests/billing-services-bill.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Billing Service - Bills tests
+
+    Background: 
+    # Calling access token
+     * def authUsername = employeeUserName
+     * def authPassword = employeePassword
+     * def authUserType = employeeType
+     * call read('../preTests/authenticationToken.feature')
+     * def billingServiceConstants = read('../constants/billing-service.yaml')
+     * def commonConstants = read('../constants/commonConstants.yaml')
+     * def consumerCode = billingServiceConstants.parameters.consumerCode
+     * def businessService = billingServiceConstants.parameters.businessService
+     * def mobileNumber = billingServiceConstants.parameters.mobileNumber
+     * def consumerCodeBill = billingServiceConstants.parameters.consumerCodeBill
+     * def invalidMobileNumber = billingServiceConstants.invalidParameters.mobileNumber
+     * def invalidConsumerCode = billingServiceConstants.invalidParameters.consumerCode
+     * def invalidBusinessService = billingServiceConstants.invalidParameters.businessService
+     * def invalidTenantId = billingServiceConstants.invalidParameters.tenantId
+     * def mustNotNullError = commonConstants.errorMessages.nullParameterError
+     * def noConsumerCodeErrorCode = billingServiceConstants.errorMessages.consumerCode.code
+     * def noConsumerCodeErrorMessage = billingServiceConstants.errorMessages.consumerCode.message
+     * def noBusinessServiceErrorCode = billingServiceConstants.errorMessages.businessService.code
+     * def noTenantIdErrorCode = billingServiceConstants.errorMessages.tenantId.code
+     * def noDemandFoundErrorCode = billingServiceConstants.errorMessages.invalidConsumerBusinessCode.code
+     * def noDemandFoundErrorMessage = billingServiceConstants.errorMessages.invalidConsumerBusinessCode.message
+     * def invalidTenantIdErrorCode = billingServiceConstants.errorMessages.invalidTenantId.code
+     * def invalidTenantIdErrorMessage = billingServiceConstants.errorMessages.invalidTenantId.message
+
+
+@fetchbill_01 @fetchbill
```

Comment:
please check and add @positive or @negative tags for other test scenarios as well

Reply:
Comment addressed